src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -190,13 +195,28 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
 
     public static struct Context
     {
+        import ocean.util.serialize.contiguous.Serializer;
+        import ocean.util.serialize.contiguous.Deserializer;
+
+        private void[] user_specified_params_blob;
```

Comment:
We thought of that. However, `Contiguous.data` is not a public field so the de/serialiser would mess with private property. Do you think that would be better?